scripts/prepdocs.py

```diff
@@ -118,7 +119,7 @@ def remove_blobs(filename):
     blob_container = blob_service.get_container_client(args.container)
     if blob_container.exists():
         if filename is None:
-            blobs = blob_container.list_blob_names()
+            blobs = iter(blob_container.list_blob_names())
```

Comment:
This function was returning an iterator type object, but it wasn't being recognized as such, so calling iter() makes it clear.